Documentation/resources.md

```diff
@@ -0,0 +1,19 @@
+## Routes
```

Comment:
@bburt-rh this is a very early draft showing how the "openshift.io/description" annotations can be translated into a MarkDown page. Feel free to edit/comment this file and I'll incorporate the changes to the source code.

Like we do for the Documentation/api.md, generating an asciidoc version shouldn't be complicated.